code/addons/docs/package.json

```diff
@@ -122,7 +122,7 @@
     "ts-dedent": "^2.0.0"
   },
   "devDependencies": {
-    "react": "^16.14.0",
+    "react": "^18.2.0",
```

Comment:
@ndelangen I don't think we have to upgrade react in docs as well if this PR intends to upgrade the manager's react version only. WDYT?

Reply:
That's just a devDependency, so it's OK, I think?